### PR TITLE
Add github issue forms

### PR DESCRIPTION
**Pull request by cpuguy83**
This uses the newer github issues forms to create bug reports and
feature requests.

It also makes it harder for people to accidentally open a blank issue
since there are required fields that must be filled in.

Example here: https://github.com/cpuguy83/dummy/issues/new/choose

### CODE REVIEW

**Comment by thaJeztah** [comment]
nice!

Gave it a quick try; looks like pasting raw `docker info` or `docker version` output doesn't put it in a "code" format; is there a way to do so? https://github.com/cpuguy83/dummy/issues/1

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: Bug report
+description: Create a report to help us improve
+title: '[Bug]: '
+labels:
+  - kind/bug
```
Comment (AkihiroSuda) [inline .github/ISSUE_TEMPLATE/bug_report.yml:5]:
```suggestion
  - kind/unconfirmed-bug-claim
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+name: Bug report
+description: Create a report to help us improve
+title: '[Bug]: '
```
Comment (AkihiroSuda) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]: Most "bug" claims are non-bug, so probably we shouldn't have this [Bug] header

**Review by AkihiroSuda — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (cpuguy83) [reply .github/ISSUE_TEMPLATE/bug_report.yml:3]: Possibly so.
Also makes sure people don't forget to add a title :)

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (cpuguy83) [reply .github/ISSUE_TEMPLATE/bug_report.yml:5]: How about I add the triage label?

**Review by samuelkarp — APPROVED** [review]
LGTM!

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**